dynamite_sdk/objects/baselines.py

```diff
@@ -0,0 +1,208 @@
+            self.external_originating_bytes_received_sum = _source['zeek'].get('out_orig_bytes_rcvd', 0)
+            self.internal_originating_packets_sent_count = _source['zeek'].get('int_orig_pkts_sent', 0)
+            self.internal_originating_packets_received_count = _source['zeek'].get('int_orig_pkts_recvd', 0)
+            self.external_originating_packets_sent_count = _source['zeek'].get('out_orig_pkts_sent', 0)
+            self.external_originating_packets_received_count = _source['zeek'].get('out_orig_pkts_recvd', 0)
+            self.smb_client_connection_count = _source['zeek'].get('smb_client_conns', 0)
+            self.smb_server_connection_count = _source['zeek'].get('smb_server_conns', 0)
+            self.smb_producer_consumer_ratio_average = _source['zeek'].get('pcr_smb_avg', 0)
+            self.smb_producer_consumer_ratio_max = _source['zeek'].get('pcr_smb_max', 0)
+            self.smb_producer_consumer_ratio_min = _source['zeek'].get('pcr_smb_min', 0)
+            self.http_client_connection_count = _source['zeek'].get('http_client_conns', 0)
+            self.http_server_connection_count = _source['zeek'].get('http_server_conns', 0)
+            self.http_producer_consumer_ratio_average = _source['zeek'].get('pcr_http_avg', 0)
+            self.http_producer_consumer_ratio_max = _source['zeek'].get('pcr_http_max', 0)
+            self.http_producer_consumer_ratio_min = _source['zeek'].get('pcr_http_min', 0)
+            self.dns_client_connection_count = _source['zeek'].get('dns_client_conns', 0)
+            self.dns_server_connection_count = _source['zeek'].get('dns_server_conns', 0)
+            self.dns_producer_consumer_ratio_average = _source['zeek'].get('pcr_dns_avg', 0)
+            self.dns_producer_consumer_ratio_max = _source['zeek'].get('pcr_dns_max', 0)
+            self.dns_producer_consumer_ratio_min = _source['zeek'].get('pcr_dns_min', 0)
+            self.ssl_client_connection_count = _source['zeek'].get('ssl_client_conns', 0)
+            self.ssl_server_connection_count = _source['zeek'].get('ssl_server_conns', 0)
+            self.ssl_producer_consumer_ratio_average = _source['zeek'].get('pcr_ssl_avg', 0)
+            self.ssl_producer_consumer_ratio_max = _source['zeek'].get('pcr_ssl_max', 0)
+            self.ssl_producer_consumer_ratio_min = _source['zeek'].get('pcr_ssl_min', 0)
+            self.rdp_client_connection_count = _source['zeek'].get('rdp_client_conns', 0)
+            self.rdp_server_connection_count = _source['zeek'].get('rdp_server_conns', 0)
+            self.rdp_producer_consumer_ratio_average = _source['zeek'].get('pcr_rdp_avg', 0)
+            self.rdp_producer_consumer_ratio_max = _source['zeek'].get('pcr_rdp_max', 0)
+            self.rdp_producer_consumer_ratio_min = _source['zeek'].get('pcr_rdp_min', 0)
+        except KeyError:
+            raise InvalidIntervalError('Invalid dns record, missing "zeek" section')
```

Comment:
Exception text refers to "Invalid dns record", should this be "Invalid observation record" or something similar?